cmd/hypper/upgrade.go

```diff
@@ -151,6 +165,13 @@ func newUpgradeCmd(cfg *action.Configuration, logger log.Logger) *cobra.Command
 				return err
 			}
 
+			// Set namespace for the upgrade client
+			if settings.NamespaceFromFlag {
+				client.Namespace = settings.Namespace()
+			} else {
+				client.SetNamespace(ch, settings.Namespace())
+			}
```

Comment:
Same here, aren't we missing setting the Config namespace with a call to `client.Config.SetNamespace(client.Namespace)`?
Analogous to: https://github.com/rancher-sandbox/hypper/blob/1a686aa3a18ebb10bbb2727baa28ea7e416ccc1a/cmd/hypper/install.go#L126

Reply:
yup yup!